badge_server/templates/one-badge.html

```diff
@@ -141,29 +143,21 @@ <h6>{{ detail.latest_version }}</h6>
                                                         </div>
                                                     </td>
                                                     <td>
-                                                       <span class="table-data__info">{{ detail.detail }}</span>
+                                                       <div class="table-data__info">
+                                                            <h6>{{ detail.detail }}</h6>
+                                                        </div>
                                                     </td>
                                                 </tr>
                                             {% endfor %}
                                         </tbody>
                                     </table>
-                                {% elif dependency_res.status.name == 'UP_TO_DATE' %}
+                                {% elif dependency_res.status.name == 'SUCCESS' %}
                                     <h4 class="title-3 m-b-30">All dependencies are up to date!</h4>
+                                {% elif dependency_res.status.name == 'OBSOLETE_DEPENDENCY' %}
+                                    <h4 class="title-3 m-b-30">{{ dependency_res.deprecated_deps }}</h4>
                                 {% endif %}
                             </div>
                         </div>
-                        {% if dependency_res.deprecated_deps | length > 0 %}
```

Comment:
I think this should not be deleted

Reply:
It used to be separated in two boxes in the UI previously as we didn't have `OBSOLETE_DEPENDENCY` status. And for now it would be strange to place the deprecated dependencies in a separate box from the dependency result box. So that I merged those two.

But if the obsolete dependency status means something else, it makes sense to keep them separated.

Reply:
`OBSOLETE_DEPEDENCY` replaced `HIGH_PRIORITY` which existed in the previous UI...

Reply:
Changed.